spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+require "users_controller"
```

Comment:
No need to manually call this line because the rails_helper loaded the whole rails environment